cloud-config-client/src/main/java/com/example/cloud/config/client/ConfigClientApplication.java

```diff
@@ -0,0 +1,41 @@
+package com.example.cloud.config.client;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import reactor.core.publisher.Mono;
+
+import org.springframework.aot.hint.MemberCategory;
+import org.springframework.aot.hint.RuntimeHints;
+import org.springframework.aot.hint.RuntimeHintsRegistrar;
+import org.springframework.aot.hint.TypeReference;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+@RestController
+public class ConfigClientApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(ConfigClientApplication.class, args);
+	}
+
+	@GetMapping("/")
+	Mono<String> test() {
+		return Mono.just("test");
+	}
+
+}
+
+class ConfigClientApplicationHints implements RuntimeHintsRegistrar {
+
+	@Override
+	public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
+		hints.reflection().registerType(TypeReference.of(ConfigClientApplication.class),
```

Comment:
Why is this necessary? Those hints should have been inferred from boot / SF.

Reply:
Righ @mhalbritter, it's unnecessary now. Will remove it.

Reply:
Done.